### PR TITLE
[caf] Update caf to 0.17.6

### PR DESCRIPTION
**Pull request by rkesters**
Update caf to 0.17.6

- What does your PR fix?  I did not create an issues, I just did the work of updating caf to the latest version. 

- Which triplets are supported/not supported? no change
- Have you updated the CI baseline? no

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)? yes

### CODE REVIEW

**Comment by rkesters** [comment]
Can I get some help from a maintainer? The one job that is failing appears ti fail ob build tinyxml, which i did not change.

File: ports/caf/portfile.cmake [hunk 1]
```diff
@@ -18,12 +18,11 @@ endif()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO actor-framework/actor-framework
-    REF b05a4b7b2a6f59b0508db84ea68e5ada2224f904 # 0.17.3
-    SHA512 02b1598f5090674bc2ce4fdbcd70a4669c8c0710e98d910dbd77338c088cc87db4e0eeb3dd89100772ff4f74bdf09a7f70b8975d79c816e422dc34a356f8ba4e
+    REF f7d4fc7ac679e18ba385f64434f8015c3cea9cb5 # 0.17.3
```
Comment (PhoebeHui) [inline ports/caf/portfile.cmake:21]:
```suggestion
    REF f7d4fc7ac679e18ba385f64434f8015c3cea9cb5 # 0.17.6
```

**Review by PhoebeHui — COMMENTED** [review]
I didn't see theat failure, it seems flaky issues, I have rerun the pipeline.

**Review by PhoebeHui — APPROVED** [review]

**Comment by rkesters** [comment]
@PhoebeHui do I need to do anything else to get this merged? If not when do you think it will be merged?

Thank you for you help.

**Comment by PhoebeHui** [comment]
@BillyONeal, could you help merge this PR?

File: ports/caf/portfile.cmake [hunk 2]
```diff
@@ -18,12 +18,11 @@ endif()
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO actor-framework/actor-framework
-    REF b05a4b7b2a6f59b0508db84ea68e5ada2224f904 # 0.17.3
-    SHA512 02b1598f5090674bc2ce4fdbcd70a4669c8c0710e98d910dbd77338c088cc87db4e0eeb3dd89100772ff4f74bdf09a7f70b8975d79c816e422dc34a356f8ba4e
+    REF f7d4fc7ac679e18ba385f64434f8015c3cea9cb5 # 0.17.6
+    SHA512 8b4719c26dfad68eed6f2528263702e42f9865bb7a9f2d40909dc6c3fc20bb7259fe44a5f89390ba714c7f9359db2d171ff44685641962c24a70f4e2aa3f3f65
     HEAD_REF master
 	PATCHES
 		openssl-version-override.patch
```
Comment (BillyONeal) [inline ports/caf/portfile.cmake:25]: Why are you deleting the fix-arm patch but not deleting the patch file?

**Review by BillyONeal — CHANGES_REQUESTED** [review]
Patch file quesiton...

File: ports/caf/portfile.cmake (hunk 2 above)
Comment (rkesters) [reply ports/caf/portfile.cmake:25]: Oversight on my part. I'll take care of it in a few minutes. Ty for the catch.

Comment (rkesters) [reply ports/caf/portfile.cmake:25]: Deleted the file and pushed.

**Review by BillyONeal — APPROVED** [review]
Sounds good; just wanted to make sure you intended to remove it :). Thanks!

**Comment by rkesters** [comment]
@PhoebeHui or @BillyONeal  can one of you help the Azure Pipeline it appears to be hung up, been running for over a day.

**Comment by BillyONeal** [comment]
It's not been over a day; I pushed a change 7 hours ago that reset it. You just got a little unlucky and there were like 3 PRs that each rebuilt the full tree in front of this one. We have 18 windows boxes in the fleet and they've been busy; rebuilding 1300+ ports takes awhile.

(Bottom 3 rows are the Linux, Windows, and Mac fleet respectively; the Windows fleet is much bigger because we test 6 triplets for it vs only 1 for the other 2 platforms
image
)

**Comment by PhoebeHui** [comment]
The CI failed to download source before, passed after rerun.

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**